vector/src/main/java/im/vector/app/core/services/CallRingPlayer.kt

```diff
@@ -120,6 +123,12 @@ class CallRingPlayerOutgoing(
         }
     }
 
+    private fun WebRtcCallManager.setAudioModeToCallType() {
+        currentCall.get()?.let {
+            audioManager.setMode(if (it.mxCall.isVideoCall) CallAudioManager.Mode.VIDEO_CALL else CallAudioManager.Mode.AUDIO_CALL)
+        }
+    }
```

Comment:
Do we need a fallback if `currentCall.get()` is null?

Maybe better to do:
```kotlin
audioManager.setMode(if (currentCall.get()?.mxCall?.isVideoCall.orFalse()) CallAudioManager.Mode.VIDEO_CALL else CallAudioManager.Mode.AUDIO_CALL)
```

Reply:
using a fallback sounds good :+1: I do wonder if it's possible to have a `null` call at this point in the flow (and if so should the entire flow throw?)

f89a2b4e56a3474c94610c35bbbf9e6092e47a40

Reply:
I do not know... I would say out of scope of this PR :/